src/lib/containers/QueryWrapper.tsx

```diff
@@ -262,6 +263,7 @@ export default class QueryWrapper extends React.Component<
       isLoading: true,
       lastQueryRequest: clonedQueryRequest,
       selectedRowIndices: [], // reset selected row indices any time the query is re-run
+      error: undefined,
```

Comment:
The error was not being reset when the query was re-executed.  We may not have known this because errors were lost in some cases!

Reply:
Good catch!